app/models/debate.rb

```diff
@@ -11,6 +13,30 @@ class Debate < ApplicationRecord
   before_validation :set_code
   after_create :create_answers
 
+  def positive_count_with_person
+    votes_count = answers.positive.first.votes.count
+    pluralize(votes_count, 'person')
+  end
```

Comment:
I would define `positive_votes` method on `Debate` and move presentation logic to some kind of presenter class. In `positive_votes` You could use another method - `positive_answer` to avoid long chaining